source/install/cloud-changelog.md

```diff
@@ -4,12 +4,54 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2023-08-31](#release-2023-08-31)
 - [Release 2023-08-09](#release-2023-08-09)
 - [Release 2023-08-03](#release-2023-08-03)
 - [Release 2023-07-26](#release-2023-07-26)
 - [Release 2023-07-20](#release-2023-07-20)
 - [Release 2023-07-19](#release-2023-07-19)
-- [Release 2023-07-11](#release-2023-07-11)
+
+## Release 2023-08-31
+
+### Improvements
+
+#### User Interface (UI)
+ - Added functionality to bulk mark a whole channel category as read.
+ - Removed Boards product tour code.
+ - Replaced Gfycat with Giphy in the gif picker.
+ - Updated Focalboard plugin version to 7.11.3.
+ - Pre-packaged Playbooks version 1.38.1.
+ - Upgraded prepackaged Zoom plugin to v1.6.2, and Antivirus plugin to v1.0.0.
```

Comment:
@amyblais I would recommend removing the reference to Antivirus for cloud-specific release notes. #24336's release note was on-prem inclusive.

```suggestion
 - Upgraded prepackaged Zoom plugin to v1.6.2.
```